docs/MPQEditor.md

```diff
@@ -0,0 +1,63 @@
+# MixedPrecisionQantizationEditor (MPQEditor for short)
+
+## About
+
+MPQEditor povides easy way of creation/editing MPQ json files, suitable for Quantization step in CfgEditor. Also it gives the ability to select the most sensitive to quantization layers using `visq` data. 
+
+## How to use
+
+Currently MPQEditor is integrated into EXPLORER view by context menu item "Create MPQ json".
+
+In order to use it:
+1) call context menu for a `.circle model` in EXPLORER view
+2) call "Create MPQ json"
+3) give a name to MPQ json
+4) edit MPQ json using MPQEditor
+5) all layers from 'layers' section in json are shown in grid-table, so that their options for quantization can be edited using their respective dropdowns
+6) to make 'model-graph-view' more informative the user may use 'visq' information, it shows which layers are more sensitive to quantization than others
+
+<img src="./images/MPQ_UI.png"/>
```

Comment:
please refine this image. do not use red color for text. use smaller size for explanation.

Reply:
Add "Default Parameters" in GUI not in this image.

Reply:
Ahhh... Sorry. I'll fix it.